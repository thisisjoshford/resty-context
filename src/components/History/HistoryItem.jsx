import React from 'react';
import PropTypes from 'prop-types';

const HistoryItem = ({ url, method }) => (
  <section>
    <p>{url}</p>
    <p>{method}</p>
  </section>
);

HistoryItem.propTypes = {
  url: PropTypes.string.isRequired,
  method: PropTypes.string.isRequired
};

export default HistoryItem;
