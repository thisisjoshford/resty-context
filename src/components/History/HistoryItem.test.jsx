import React from 'react';
import { shallow } from 'enzyme';
import HistoryItem from './HistoryItem';

describe('History Item component', () => {
  it('renders History Item', () => {
    const wrapper = shallow(
      <HistoryItem 
        url="http://www.randomapi.com"
        method="GET"
      />);
    expect(wrapper).toMatchSnapshot();
  });
});
