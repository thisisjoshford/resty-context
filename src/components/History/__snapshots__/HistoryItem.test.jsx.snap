// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History Item component renders History Item 1`] = `
<section>
  <p>
    http://www.randomapi.com
  </p>
  <p>
    GET
  </p>
</section>
`;
